2018/python/rules_engine.py

```diff
@@ -635,7 +635,11 @@ def v628_1(self):
 		unless an ethnicity is provided in Ethnicity of Applicant or Borrower: Free Form Text Field for Other Hispanic or Latino."""
 		field = "app_eth_1"
 		edit_name = "v628_1"
-		fail_df = self.lar_df[~(self.lar_df.app_eth_1.isin(("1","11", "12", "13", "14", "2", "3","4")))|((self.lar_df.app_eth_free=="")&(self.lar_df.app_eth_1==""))]
+		fail_df = self.lar_df[((~self.lar_df.app_eth_1.isin(("1","11", "12", "13", "2", "3","4"))) & 
+								(self.lar_df.app_eth_free == ''))|
+							  ((self.lar_df.app_eth_1.isin(("1","11", "12", "13", "2", "3","4"))) & 
+								(self.lar_df.app_eth_free != ''))|
```

Comment:
I don't think v628_1 actually checks the second two conditions you've implemented. 

Is there a reason we would need these even though they're not in the FIG? My thought is that we need to match the FIG when producing test fails so we can accurately assess whether the platform conforms to the FIG.

I think they're interesting for completeness sake and I'll make a note for FIG 2020 on this.

Reply:
Sounds good. I added commits that would remove some conditions. Thanks!